.github/marquee.js

```diff
@@ -0,0 +1,38 @@
+async function createMarquee() {
```

Comment:
nice work here with the marquee. There is one important issue you should notice - the marquee is widing (making the width bigger) the window, and you should prevent it.